addons/vehicles/XEH_postInit.sqf

```diff
@@ -2,20 +2,31 @@
 #include "script_component.hpp"
 
 if (!hasInterface) exitWith {};
-
+GVAR(isSpeedLimiter) = false;
 // Add keybinds
 ["ACE3 Vehicles", QGVAR(speedLimiter), localize LSTRING(SpeedLimiter),
 {
-    // Conditions: canInteract
-    if !([ACE_player, objNull, ["isnotinside"]] call EFUNC(common,canInteractWith)) exitWith {false};
-    // Conditions: specific
-    if !(ACE_player == driver vehicle ACE_player &&
-    {vehicle ACE_player isKindOf 'Car' ||
-        {vehicle ACE_player isKindOf 'Tank'}}) exitWith {false};
+    private _connectedUAV = getConnectedUAV ACE_player;
+    private _uavControll = UAVControl _connectedUAV;
+    if ((_uavControll select 1) == "DRIVER") then {
+        if !(_connectedUAV isKindOf "UGV_01_base_F") exitWith {false};
```

Comment:
`isUav = 1;`

Not many UAV's that are also a car/tank; so hard coded isn't terrible (this is a pretty odd edge case)